config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  TOKEN: "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",
```

Comment:
Don't include your token. Make sure you generate a new one if you accidentally push it.

Reply:
Oh whoops okay will create a new token